logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/usecase/GetIncomingCallsUseCase.kt

```diff
@@ -51,7 +51,9 @@ internal class GetIncomingCallsUseCaseImpl(
             .flatMapLatest {
                 //if user is AWAY we don't show any IncomingCalls
                 if (it.availabilityStatus == UserAvailabilityStatus.AWAY) flowOf(listOf())
-                else callRepository.incomingCallsFlow()
+                else callRepository.incomingCallsFlow().distinctUntilChanged {
+                        old, new -> old.firstOrNull()?.conversationId == new.firstOrNull()?.conversationId
```

Comment:
Aren't both conversationId from the `old` and the `new` always the same if we are receiving a call from the same conversation? 🤔

Reply:
yes right, but they might have different participants..That't why `distinctUntilChanged()` will not work and we need to check by `conversationId`